downgrade

```diff
@@ -127,14 +133,19 @@ sort_packages() {
 output_package() {
   local pkg indicator=" "
 
-  for pkg in "${installed[@]}"; do
-    case "$2" in
-      *$pkg*)
-        indicator="*"
+  if [[ ! -z $current ]] && [[ $2 == *"$current"* ]]; then
```

Comment:
Please use `-n` for `! -z`.

I would quote `"$current"` and `"$2"` even though I don't think it's strictly needed (at least, ShellCheck didn't complain) because you're in `[[` and not `[`.